docs/docker-image-details.md

```diff
@@ -112,28 +112,32 @@ You can use these kinds of Docker images to facilitate integration testing of al
 You can use this kind of image to test the catalog of Apache Hive with kerberos enable
 
 Changelog
-- gravitino-ci-kerberos-hive:0.1.5
+
+- apache/gravitino-ci:kerberos-hive-0.1.5 (Switch to Apache official DockerHub repository)
```

Comment:
Maybe we would better use 0.1.6 here.

Reply:
We can keep the current version as it is because they are now different Docker repos.